projects/step-frontend/src/lib/modules/timeseries/components/chart-dashlet-settings/chart-dashlet-settings.component.html

```diff
@@ -2,10 +2,26 @@
   <div step-modal-window-body>
     <form #formContainer="ngForm">
       <ng-container *ngIf="item">
-        <step-form-field [showRequiredMarker]="true">
-          <step-label>Name</step-label>
-          <input placeholder="Dashlet name" [(ngModel)]="item.name" name="name" required />
-        </step-form-field>
+        <div class="form-row">
```

Comment:
It makes sense to apply the modern template syntax instead of *ngIf and *ngFor directives